packages/UI/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,45 @@
+import { tv } from 'tailwind-variants';
+import Checkmark from './../../assets/icons/Checkmark';
+import type { CheckboxProps } from '.';
+
+export default function Checkbox({
+  checked = false,
+  className,
+  state,
+  ...props
+}: CheckboxProps) {
+  const checkbox = tv({
+    base: 'flex h-6 w-6 items-center justify-center rounded-sm border-2 border-solid border-brand-alt-nebula-500 bg-grayscale-50',
+    variants: {
+      state: {
+        focus:
+          'outline outline-2 outline-offset-2 outline-brand-alt-nebula-500',
+        hover:
+          'outline outline-4 outline-offset-0 outline-brand-alt-nebula-100',
+        disable: 'border-grayscale-500 bg-grayscale-200',
+      },
```

Comment:
These states can be handled better using [tailwind classes](https://tailwindcss.com/docs/hover-focus-and-other-states),
the current approach means the end user won't be able to toggle the hover state nor the focus state using their mouse/tab key for example.